### PR TITLE
ShadowrunAnarchy - new translatable items (with French translation)

### PR DESCRIPTION
**Pull request by Namergon**
## Changes / Comments

New translation related things in the ShadowrunAnarchy character sheet:
1. Skill section: the linked attribute initial is now translatable. 
French the initial is now in French for the French character sheet. See fr.json to apply for other languages.
- Files: ShadowrunAnarchy.html, fr.json

2. Weapons section: "DAM" label is now translatable.
French character sheet now displays "VD".
- Files: ShadowrunAnarchy.html, fr.json

3. Misc. minor typo fixes
Minor typos in some French texts (fr.json)
1 comment typo in ShadowrunAnarchy.css

## Roll20 Requests

Comments are very helpful for reviewing the code changes. Please answer the relevant questions below in your comment.

- [yes ] Does the pull request title have the sheet name(s)? Include each sheet name.
- [no ] Is this a bug fix?
- [no ] Does this add functional enhancements (new features or extending existing features) ?
- [ no] Does this add or change functional aesthetics (such as layout or color scheme) ? 
- [ ] If changing or removing attributes, what steps have you taken, if any, to preserve player data ?
- [ ] If this is a new sheet, did you follow [Building Character Sheets standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) ?

If you do not know English. Please leave a comment in your native language.

### CODE REVIEW

**Comment by rpelisse** [comment]
@clevett As you are the original author of the sheet, maybe you want to take a look? BTW, thanks to the quarantine a lot of french player have started using this character sheet and it has been praised for its quality! (I though you may appreciate the nice "ego stroking" from over the Atlantic ;) )

**Comment by clevett** [comment]
Appreciate the heads up and the changes look good. The French version of the paper sheet is sooo nice.

File: ShadowrunAnarchy/translations/fr.json [hunk 1]
```diff
@@ -90,5 +90,12 @@
     "whisper": "Murmurer",
     "wil": "VOL",
     "willpower": "Volonté",
-    "wounds": "Blessures"
+    "wounds": "Blessures",
+	"+S": "+F",
```
Comment (clevett) [inline ShadowrunAnarchy/translations/fr.json:94]: These new keys also need to be added to the translation.json file in the top folder. Just adding them to the fr.json file will result in the other languages missing them and breaking for the sheet for those languages.

Comment (Namergon) [reply ShadowrunAnarchy/translations/fr.json:94]: I should have thought about it, indeed.
Done!

**Pull request merged**